app/models/package.scala

```diff
@@ -5,7 +5,7 @@ package object models {
   import play.api.Logger
   import play.api.Play.current
 
-  lazy val db = {
+  def db = {
```

Comment:
Yeah, this change was required on my machine to get the tests to run. Otherwise, I get this error:

```
[error] Caused by java.sql.SQLException: Attempting to obtain a connection from a pool that has already been shutdown.
```

I'll take a look at getting the tests to run against either an in-memory db or a different postgres db.